metpy/calc/cross_sections.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) 2017 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains calculation of various derived indices.
```

Comment:
Docstring should probably say something about cross-sections. Also the copyright year should be 2018.